incubator/faucet/client/cli/query.go

```diff
@@ -30,36 +19,30 @@ func GetQueryCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	pooltoyQueryCmd.AddCommand(
-		flags.GetCommands(
-			GetCmdWhenBrrr(queryRoute, cdc),
-		)...,
-	)
+	pooltoyQueryCmd.AddCommand([]*cobra.Command{
+		queryWhenBrrr(),
+	}...)
 
 	return pooltoyQueryCmd
 }
 
-func GetCmdWhenBrrr(queryRoute string, cdc *codec.Codec) *cobra.Command {
+func queryWhenBrrr() *cobra.Command {
```

Comment:
nit: I think the original function name is more consistent with the other cli command functions. wdyt?

Reply:
I think the original function name is confusing bc it's too semantically similar to the root query module GetQueryCmd, I'd like to indicate that this is a subquery

Reply:
I just noticed that is the pattern in most modules I've seen e.g -> 
 
https://github.com/PeggyJV/sommelier/blob/main/x/il/client/cli/query.go#L34
I think it's GetCmd... as each of those functions are just getters for the actual cobra.Command.